configs/setups/awicm3/awicm3.yaml

```diff
@@ -294,13 +299,21 @@ fesom:
         # Sets kkp_kv0 in namelist.cvmix to 5e-2 for the 1.0-deck version to be able to reproduce
         # old experiments. 5e-3 is a better value so it is used for XIOS version
         choose_general.version:
+                "1.0-master":
+                        model_tag: "awicm-3-master"
+                        # namelist_dir: "${model_dir}/fesom-2.0-master/config/" TODO: this is the line we actually want to use
```

Comment:
`model_dir` during run time refers to the folder of that particular component (i.e. `model_dir` in this case should be `<path_to_model_folder>/awicm3-1.0-master/fesom-2.0-master`), each component has its own `model_dir`. So I would suggest to write this:
```
namelist_dir: "${model_dir}/config/"
```